### ISSUE TITLE
Canvas initiation with constraints

### ISSUE THREAD

**Issue by procop95** [freegroup/draw2d#29]
Dears.

this issue is related to issue #7 opened by me previously. 
problem is: in cases when canvas is instantiated by calling
`canvas = new draw2d.Canvas("canvas", 4000, 4000) - without directly mentioning css {width: 4000px; height: 4000px}`  in container;
will cause figures not be able to be moved outside initial area (viewport) of the container.
 
this behaviour is caused by line 121 of Canvas.js:
```js
// installed to all added figures to avoid that a figure can be placed outside the canvas area
// during a drag&drop operation
this.regionDragDropConstraint = new draw2d.policy.figure.RegionEditPolicy(0, 0, this.getWidth(), this.getHeight())
````

where `this.getWidth()` and `this.getHeight()` will return initial viewport dimensions.

change this to:
```js
   this.regionDragDropConstraint = new draw2d.policy.figure.RegionEditPolicy(0, 0, this.initialWidth, this.initialWidth)
```
will do proper job, if you agree.

thanks
Alex

**Comment by InannaInai** [comment]
You have an error in your solution:
I think it should be:
`  this.regionDragDropConstraint = new draw2d.policy.figure.RegionEditPolicy(0, 0, this.initialWidth, this.initialHeight)`

**Comment by fmaiabatista** [comment]
This fix (last arg corrected to `this.initialHeight`) solved my issue. This line was also causing scrolling to malfunction.

**Comment by freegroup** [comment]
fixed with version 1.0.23. the width/height in the constructor is propagated as CSS attribute to the named DIV
